src/documents/blog/09-msghero-introduction-goals.html.md

```diff
@@ -20,6 +22,10 @@ I moved on to Haxe when I learned I could get HTML5 and CPP builds for (almost)
 
 Flixel is great for me because I no longer have to even think about updating and rendering. Input is taken care of; spritesheets are handled. Life is good.
 
+I'm also helping with a UI library called [HaxeUI](https://github.com/haxeui/haxeui-core) that I'll be talking about in the future. Stay tuned for that one.
+
+![](/images/blog/haxeui.png)
```

Comment:
I grabbed that one from the website but I can just compile it in flixel and screenshot that